armada-ths/ais
# Recruitment application fix

Fixed recruitment application view and wrote a test to catch similar problems in the future

to test this:
* login on a user that can apply for recruitment (non-superuser)
* goto localhost:8000/fairs/2017/recruitment/n/application/new
* make sure it doesn't give you an error
